File: Data Structures & Algorithms/remove-duplicates-from-sorted-array-ii/submission-0.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        L=0
        occur=0
        for R in range(1,len(nums)):
            if (nums[R]!=nums[R-1]):
                L+=1
                nums[L]=nums[R]
                occur=0
            elif (nums[R]==nums[R-1] and  occur==0):
                L+=1
                nums[L]=nums[R]
                occur+=1
            else:
                occur+=1
        return L+1
        

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.go
func topKFrequent(nums []int, k int) []int {
   freq := make(map[int]int)
   for _, num := range nums{
		val,ok := freq[num]
		if !ok {
			freq[num]=1
		} else {
			freq[num]=val+1
		}
   }
   type Pair struct {
	 num int
	 count int
   }
   var occur []Pair
   for num,count := range freq {
	occur = append(occur,Pair{num,count})
   }
   sort.Slice(occur, func(i, j int) bool {
    return occur[i].count > occur[j].count
	})
	var res []int
	for i:=0;i<k;i++ {
		res = append(res,occur[i].num)
	}
	return res

}
